### DISCUSSION TITLE
请问如何修改model的检查点文件以及日志文件的保存路径

### DISCUSSION THREAD

**Discussion by luoz66** [open-mmlab/mmdetection#9270] [Q&A] [answered]

**Comment by mm-assistant** [comment]
We recommend using English or English & Chinese for issues so that we could have broader discussion.

**Comment by jbwang1997** [comment]
You can change your work dir by add `work_dir` in your config or add `--work-dir` in command line.

**Comment by luoz66** [comment]
> You can change your work dir by add `work_dir` in your config or add `--work-dir` in command line.

我的意思是检查点和日志文件自己指定到不同的目录，分开放

**Comment by ZwwWayne** [comment, marked as answer]
you can set out_dir in the checkpoint hook

**Reply by ZwwWayne** [reply]
See https://github.com/open-mmlab/mmcv/blob/master/mmcv/runner/hooks/checkpoint.py#L56.
Therefore, you can set in the `checkpoint_config` in the config
